src/libraries/System.Net.WebSockets.Client/tests/SendReceiveTest.Http2.cs

```diff
@@ -0,0 +1,109 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Linq;
+using System.Net.Http;
+using System.Net.Sockets;
+using System.Net.Test.Common;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Xunit;
+using Xunit.Abstractions;
+
+using static System.Net.Http.Functional.Tests.TestHelper;
+
+namespace System.Net.WebSockets.Client.Tests
+{
+    public abstract class SendReceiveTest_Http2 : ClientWebSocketTestBase
```

Comment:
I think it should not be abstract?

```suggestion
    public class SendReceiveTest_Http2 : ClientWebSocketTestBase
```

Reply:
Right, sorry 😅